build/buildRunTimeImages.sh

```diff
@@ -64,6 +64,7 @@ fi
 for generateDockerFile in $generateDockerFiles; do
     echo
     echo "Executing '$generateDockerFile'..."
+    chmod +x "$generateDockerFile"
```

Comment:
is still this needed, even after that `generateDockerfiles.sh` was given the correct permissions?

Reply:
I kept it for safety, in case it happens again it's hard to find compared to other scripts, what do you think?